Makefile

```diff
@@ -21,6 +21,24 @@ BENCH_RUN=NONE
 test:
 	$(GOTEST) -run=$(TEST_REGEX) -count=$(TEST_COUNT) ./...
 
+# lint runs all local linters that must pass before pushing
+.PHONY: lint lint-install lint-fast
+lint: lint-install
+	goimports -w .
```

Comment:
`goimports` and `gofmt` are already part of `golangci-lint` does it make sense to run it separately?

https://golangci-lint.run/usage/linters/

You would reduce amount with 3rd party requirements when using only `golangci-lint` and you would reduce setup time, locally and in CI.

Reply:
It makes sense, yes, but with some more concerns on the config. Will be a separate PR adding goci lint.